src/validator/pattern/pattern.ts

```diff
@@ -1,4 +1,4 @@
-import {STOpIsHexColorCode, createIsHexColorCode} from './hex-color-code';
+import {STOpIsHexColorCode, createIsHexColorCode} from './is-hexColorCode';
```

Comment:
Since we've renamed the is-hexColorCode file to is `is-hex-color-code` remember to update the import here.